pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: pana
 description: Evaluate the health and quality of a Dart package
-version: 0.11.8
+version: 0.11.9-dev
```

Comment:
I don't think we'll have 0.11.9, I'm targeting 0.12.0 with breaking changes.

Reply:
Ack. Feel free to run over it...